camel/toolkits/openai_function.py

```diff
@@ -11,6 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
```

Comment:
Do we need to also change the name of this file, such as `function_tool.py`?

Reply:
updated